src/coll/mod.rs

```diff
@@ -276,52 +277,26 @@ impl Collection {
         Ok(doc)
     }
 
-    // Helper method for validated replace and update commands.
-    fn find_one_and_replace_or_update(&self,
```

Comment:
After some refactoring to pass the options document directly around, this method isn't necessary anymore